octodns/provider/base.py

```diff
@@ -55,14 +59,19 @@ def raise_if_unsafe(self):
             update_pcent = self.change_counts['Update'] / existing_record_count
             delete_pcent = self.change_counts['Delete'] / existing_record_count
 
-            if update_pcent > self.MAX_SAFE_UPDATE_PCENT:
+            self.log.debug('raise_if_unsafe: update_pcent_threshold=%d, '
+                           'delete_pcent_threshold=%d',
+                           self.update_pcent_threshold,
+                           self.delete_pcent_threshold)
```

Comment:
I think it'd be preferable to log these when the provider is created, __init__ https://github.com/github/octodns/pull/120/files#diff-f1e7f60f6ff68ac72db8613d4f410dbbR95, since that's the level at which things will be configured. It'd be hard to trace back this log line to which provider emitted it at the plan point of things since the plan doesn't know it's source provider and there many plans, one per zone.